database/factories/BioFactory.php

```diff
@@ -15,4 +15,25 @@ public function definition()
             'body' => $this->faker->sentence(),
         ];
     }
+
+    public function user($user)
```

Comment:
I think this can be removed in favor of calling `Bio::factory()->for($user)` and letting the model take care of the keys.